win32/src/async_socket_win32.c

```diff
@@ -98,6 +99,10 @@ static VOID WINAPI on_io_complete(PTP_CALLBACK_INSTANCE instance, PVOID context,
             {
                 uint32_t bytes_send = (uint32_t)number_of_bytes_transferred;
 
+#ifdef ENABLE_SOCKET_LOGGING
+                LogVerbose("Asynchronous send of %" PRIu32 " bytes completed at %lf", bytes_send, timer_global_get_elapsed_us());

```

Comment:
>LogVerbose("Asynchronous send of %" PRIu32 " bytes completed at %lf", bytes_send, timer_global_get_elapsed_us()); [](http://example.com/codeflow?start=16&length=113)

You consider adding a GUID in the io_context to serve as a unique indicator of "who the packet was".

Right now, it is "size" based. 

./consider.

Reply:
Considered it. For now this was enough for latency debugging. And I'm trying to check this code in so that we don't have to write it again next time.
When the need arises for more debugging and we need to we'll add the GUID also.

---
In reply to: [654742809](https://github.com/Azure/c-pal/pull/95#discussion_r654742809) [](ancestors = 654742809)